From e8209e7b221a4a108c132fa3a8ad27e9a8018a6d Mon Sep 17 00:00:00 2001
From: Owen W. Taylor <otaylor@fishsoup.net>
Date: Tue, 10 Jan 2012 00:03:07 +0000
Subject: Handle rename of GTK+'s window-unfocused state to backdrop

GTK_STATE_FLAG_WINDOW_UNFOCUSED was renamed to
GTK_STATE_FLAG_BACKDROP.
---
diff --git a/src/ui/theme.c b/src/ui/theme.c
index 6e15d21..604505f 100644
--- a/src/ui/theme.c
+++ b/src/ui/theme.c
@@ -6403,8 +6403,8 @@ meta_gtk_state_from_string (const char *str)
     return GTK_STATE_FLAG_INCONSISTENT;
   else if (g_ascii_strcasecmp ("focused", str) == 0)
     return GTK_STATE_FLAG_FOCUSED;
-  else if (g_ascii_strcasecmp ("window-unfocused", str) == 0)
-    return GTK_STATE_FLAG_WINDOW_UNFOCUSED;
+  else if (g_ascii_strcasecmp ("backdrop", str) == 0)
+    return GTK_STATE_FLAG_BACKDROP;
   else
     return -1; /* hack */
 }
@@ -6428,8 +6428,8 @@ meta_gtk_state_to_string (GtkStateFlags state)
       return "INCONSISTENT";
     case GTK_STATE_FLAG_FOCUSED:
       return "FOCUSED";
-    case GTK_STATE_FLAG_WINDOW_UNFOCUSED:
-      return "WINDOW_UNFOCUSED";
+    case GTK_STATE_FLAG_BACKDROP:
+      return "BACKDROP";
     }
 
   return "<unknown>";
--
cgit v0.9.0.2
